script.js

```diff
@@ -0,0 +1,154 @@
+        intelligence: Math.floor(Math.random()*9),
+        courage: 10,
+    }
+}
+
+const dumbledore ={
+    name: 'Albus Dumbledore',
+    imagem: './dumbledore.png',
+    atributos:{
+        magic: Math.floor(Math.random()*9),
+        friendship: Math.floor(Math.random()*9),
+        intelligence: 10,
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const hagrid ={
+    name: 'Rubeus Hagrid',
+    imagem: './hagrid.png',
+    atributos:{
+        magic: Math.floor(Math.random()*9),
+        friendship: 10,
+        intelligence: Math.floor(Math.random()*9),
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const hermione ={
+    name: 'Hermione Granger',
+    imagem: './hermione.png',
+    atributos:{
+        magic: Math.floor(Math.random()*9),
+        friendship: Math.floor(Math.random()*9),
+        intelligence: 10,
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const neville ={
+    name: 'Neville Longbottom',
+    imagem: './neville.png',
+    atributos:{
+        magic: Math.floor(Math.random()*9),
+        friendship: Math.floor(Math.random()*9),
+        intelligence: Math.floor(Math.random()*9),
+        courage: 10,
+    }
+}
+
+const rony ={
+    name: 'Rony Weasley',
+    imagem: './rony.png',
+    atributos:{
+        magic: Math.floor(Math.random()*9),
+        friendship: 10,
+        intelligence: Math.floor(Math.random()*9),
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const sirius ={
+    name: 'Sirius Black',
+    imagem: './sirius.png',
+    atributos:{
+        magic: 10,
+        friendship: Math.floor(Math.random()*9),
+        intelligence: Math.floor(Math.random()*9),
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const snape ={
+    name: 'Severus Snape',
+    imagem: './snape.png',
+    atributos:{
+        magic: 10,
+        friendship: Math.floor(Math.random()*9),
+        intelligence: Math.floor(Math.random()*9),
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const voldemort ={
+    name: 'He-Who-Must-Not-Be-Named',
+    imagem: './voldemort.png',
+    atributos:{
+        magic: 10,
+        friendship: Math.floor(Math.random()*9),
+        intelligence: Math.floor(Math.random()*9),
+        courage: Math.floor(Math.random()*9),
+    }
+}
+
+const cartas = [harry, bellatrix, dobby, draco, dumbledore, hagrid, hermione, neville, rony, sirius, snape, voldemort]
+
+//Play:
+
+const seeYourCard = ()=>{
+    let numCardPc = Math.floor(Math.random()*cartas.length)
+    let cardPc = cartas[numCardPc]
+    cartas.slice(numCardPc, 1)
+
+    let numCardPlayer = Math.floor(Math.random()*cartas.length)
+    let cardPlayer = cartas[numCardPlayer]
+    cartas.slice(numCardPlayer, 1)
+    //console.log(cardPlayer)
+
+    document.getElementById(myCardImg).src = cardPlayer[imagem] //NOT WORKING
+    document.getElementById(myCardName) = cardPlayer[name] //NOT WORKING
+    document.getElementById(myCardAttrib) = cardPlayer[atributos] //NOT WORKING
```

Comment:
Here if you want the attributes text to show, we need to change some things:
- First you need to change to: document.getElementById(myCardAttrib).innerHTML in order to change the text inside the <p> element
- But also cardPlayer.atributos is an object with several properties, this is not a string, so we can not display an object just like that. We would need to access each attribute one by one in order to display their value:
Example: cardPlayer.atributos.magic
This will show the value they have for magic, and so on